lib/openshift/pod.rb

```diff
@@ -113,6 +114,18 @@ def ip(user: nil, cached: true, quiet: false)
       return get_cached_prop(prop: :ip, user: user, cached: cached, quiet: quiet)
     end
 
+    def ip_v6(user: nil, cached: true, quiet: false)
+      ips = get_cached_prop(prop: :ips, user: user, cached: cached, quiet: quiet)
+      ipv6 = ips.find { |ip| ip["ip"].include? ":" }
```

Comment:
Should we also check for presence of `[]` in the address? Usually IP6 addr has `[]` right?

Reply:
IPv6 has `[]` only for URLs, that is why `ip_url` method checks.

```ruby
      return raw_ip.include?(":") ? "[#{raw_ip}]" : raw_ip
```

we decided on the `_url` suffix for methods that return IPs for use in URLs.

Reply:
> Should we also check for presence of `[]` in the address? Usually IP6 addr has `[]` right?

ipv6 do not have `[]`, However if we want to access it, it should be like `curl [fd01:0:0:3::5e]:8080`